packages/peregrine/lib/targets/TalonWrapperConfig.js

```diff
@@ -33,6 +33,50 @@ class TalonWrapperConfig {
              */
             useApp: new WrappableTalon(addTransforms, 'App/useApp')
         };
+        /**
+         * Provides access to the talon used in Venia's CheckoutPage UI component.
+         * @instance
+         */
+        this.CheckoutPage = {
+            /**
+             * useCheckoutPage {@link WrappableTalon} object for the `useCheckoutPage()` talon
+             * @type {WrappableTalon}
+             */
+            useCheckoutPage: new WrappableTalon(addTransforms, 'CheckoutPage/useCheckoutPage')
+        };
+        /**
+         * Provides access to the talon used in Venia's MiniCart UI component.
+         * @instance
+         */
+        this.MiniCart = {
```

Comment:
Since we're not wrapping legacy talons you can just remove this object.

Reply:
@sirugh Isn't this the new minicart though?

Reply:
Doh - you're right @jimbo. My mistake. Forgot we moved the legacy one to `LegacyMiniCart` :P